aerospike_helpers/metrics/__init__.py

```diff
@@ -90,6 +106,58 @@ class Cluster:
     pass
 
 
+# as_node_stats has a reference to the corresponding as_node object
+# Here, we are using specific as_node fields to identify that as_node instead of storing the full as_node.
+# Since as_node has a ton of fields, we don't want to return the whole as_node.
+#
+# We also don't want to have a reference to a Node class instance
+# because our Node class has fields we don't want to expose when returning ClusterStats to the user
+# i.e Node's namespace metrics when extended metrics is disabled.
```

Comment:
Do we need this comment? I don't think it will make sense to users.

Reply:
I wanted to explain why the API for `NodeStats` was designed this way, and I figured that the first place that Python client developers would look is where the API is defined (I think it might be harder to find this comment in the Python client's C code)